db/migrate/20150102194618_remove_default_avatar_again.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveDefaultAvatarAgain < ActiveRecord::Migration
+  def change
+ 	  change_column :users, :avatar, :string, default: nil
```

Comment:
Github seems to be spacing this differently than we are.  I can't seem to get it to change.